en/tools/config-catalog-generator-cc/data/enforcer/configs.json

```diff
@@ -0,0 +1,721 @@
+              "description": "Private key path used for JWT generation."
+            },
+            {
+              "name": "validityPeriod",
+              "type": "integer",
+              "required": false,
+              "default": "3600",
+              "description": "Validity period of the JWT in seconds."
+            },
+            {
+              "name": "caCertFile",
+              "type": "string",
+              "required": false,
+              "default": "/home/wso2/security/truststore/consul/consul-agent-ca.pem",
+              "description": "This is the optional path to the CA certificate used for Consul communication, defaults to the system bundle if not specified."
+            },
+            {
+              "name": "certFile",
+              "type": "string",
+              "required": false,
+              "description": "This is the optional path to the certificate for Consul communication. If this is set, then you also need to set keyFile."
+            },
+            {
+              "name": "keyFile",
+              "type": "string",
+              "required": false,
+              "description": "This is the optional path to the private key for Consul communication. If this is set, then you need to also set certFile."
+            }
+          ]
+        }
+      ],
+      "exampleFile": "jwt_issuer.toml"
+    },
+    {
+      "title": "JWT Issuer User",
+      "options": [
+        {
+          "name": "jwtIssuer.jwtUser",
+          "required": true,
+          "description": "Configure allowed users to use the JWT generated by the Choreo Connect. You can provide multiple users.",
+          "params": [
+            {
+              "name": "username",
+              "type": "string",
+              "required": true,
+              "default": "admin",
+              "description": "Username"
+            },
+            {
+              "name": "password",
+              "type": "string",
+              "required": true,
+              "default": "$env{enforcer_admin_pwd}",
+              "description": "Password"
+            }
+          ]
+        }
+      ],
+      "exampleFile": "jwt_issuer_user.toml"
+    },
+    {
+      "title": "Auth Header",
+      "options": [
+        {
+          "name": "security.authHeader",
+          "required": false,
+          "description": "Configurations related to Authorization header",
+          "params": [
+            {
+              "name": "enableOutboundAuthHeader",
+              "type": "boolean",
+              "required": true,
+              "default": "false",
+              "description": "Remove authorization header from backend request"
+            },
+            {
+              "name": "authorizationHeader",
+              "type": "string",
+              "required": true,
+              "default": "authorization",
+              "description": "Header name which authorization token coming from the downstream client"
```

Comment:
Is this description grammatically correct? `... which authorization token ... `?